index.html

```diff
@@ -274,7 +275,11 @@ <h2 class="connect">
                 <hr />
             </div>
         </section>
+        <!-- Modal -->
+        <div id="myModal" class="modal">
 
+        </div>
```

Comment:
![Screenshot](https://user-images.githubusercontent.com/57562869/151124370-19d4d7a9-9ade-45f3-95dc-85dcf0c4e699.png)

- [OPTIONAL ] Kindly Add `background-color `to the left and right side of your images and make sure that the whole size image `width` is displayed correctly 

Figma: 

![fig](https://user-images.githubusercontent.com/57562869/151125509-3ba23570-eaee-462c-b18f-3aa1322c3a49.png)